src/www/views/servers_view/index.ts

```diff
@@ -85,20 +85,22 @@ Polymer({
         border-top-style: solid;
       }
       paper-button {
-        width: 192px;
-        height: 192px;
         display: flex;
         flex-direction: column;
         text-transform: none;
         outline: none; /* Remove outline for Safari. */
       }
+      paper-button server-connection-indicator {
+        width: 192px;
+        height: 192px;
+      }
     </style>
     <div class="server-list-container">
       <div class="flex-column-container" hidden$="[[!shouldShowZeroState]]">
         <div class="flex-column-container">
           <paper-button noink="" on-tap="_requestPromptAddServer">
             <server-connection-indicator
-              connection-state="INITIAL"
```

Comment:
Why is this changing from initial? It was a grey disk before, I think?

Reply:
i'm not exactly sure what was going on here - yes it was a grey disk, but the original `initial_disc.png` was not grey, it was a translucent blue. I can make the initial state also grey if you like and change it back, but I'm unsure what the original intent was.

Reply:
Is anything broken? Do you mean we may not be showing what was intended?
Can you show what it looks like now?
I personally think the green is confusing and I don't see anything wrong with the grey.
But @cjhenck should decide

Reply:
This was the fix I made some time ago: https://github.com/Jigsaw-Code/outline-client/pull/1155
The gray disc was what we were using and that was intentional. Perhaps we need to update the image names to avoid confusion.

Reply:
Ah, I've deleted that old image I mentioned. I agree that the green was confusing. Here's what it looks now that I've set it to "disconnected" -

![Screen Shot 2022-05-04 at 10 28 34](https://user-images.githubusercontent.com/3759828/166703208-d02005ef-ba7d-4818-a0fe-4d70539139ac.png)

Reply:
This looks different from what it was:
![image](https://user-images.githubusercontent.com/113565/167173151-1cf8b4f7-2e65-4eb9-ad92-c09618473c8c.png)

Why is it changing?

Reply:
Well, no reason, but if it matters that much to you I can make it exactly like it was before. Seems flipped and with a different gradient.